apiserver/facade/interface.go

```diff
@@ -319,3 +331,42 @@ const (
 	// established for use in the keymanager facade.
 	HTTPClientPurposeUserSSHImport HTTPClientPurpose = "ssh-key-import"
 )
+
+// ModelProviderFactory allows instantiation of a provider for the given model.
```

Comment:
We aren't instantiating a provider but just returning the provider for the model.